sweep/sweeper.go

```diff
@@ -505,6 +505,9 @@ func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch) {
 				log.Debugf("Already pending input %v received",
 					outpoint)
 
+				// Update sweep parameters.
+				pendInput.params = input.params
```

Comment:
How is this different than changing `UpdateParams` instead update the exclusive param?

Reply:
That would break the external interface of `bumpfee`. The current contract is that `bumpfee` leaves the exclusive group alone.

Reply:
We could add an option to UpdateParams to change/remove the exclusive group, and default to lave it alone. That being said, it is not a big deal, and I'm okay with this approach.

Reply:
We also need to clear the `UnconfParent` in the next PR. It worked out easier by just reoffering the whole input.